### PR TITLE
Removing words in Spanish

### PR DESCRIPTION
**Pull request by fioalvsera**
Changing Spanish words to English, to keep everything in the same language

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
-//node index.js cylinder [radio] [altura]
-//node index.js square [lado]
+//node index.js cylinder [radius] [height]
+//node index.js square [side]
 
 var firstNum = process.argv[2]
```
Comment (jseravalli) [inline index.js]: Maybe we can move the reading of the arguments inside of the if statements with more precise names. 

tip: You can use `let`

**Review by jseravalli — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (fioalvsera) [reply index.js]: Ok, I'll do that

Comment (jseravalli) [reply index.js]: No worries I'll do it myself! :)

Comment (fioalvsera) [reply index.js]: I already did it

**Comment by fioalvsera** [comment]
@jseravalli can you review it again, please

**Review by jseravalli — APPROVED** [review]
Looks good! :) good job

**Pull request merged**